js/tests.js

```diff
@@ -15,6 +15,7 @@ $(document).ready(function() {
   assertScoreByCode('24,53+53:24', 26, '2 Dwarvish Infantries should penalty eachother');
   assertScoreByCode('9,12,16,37+9:16', 95, 'Island can be used even when blanked');
   assertScoreByCode('10,53+53:10', 23, 'Elementals count their Doppelgänger');
+  assertScoreByCode('26,46,47,51,53+51:46,53:46', 60, 'Collector scores duplicated cards');
```

Comment:
Assuming this PR is closed without merging, it may be worth adding a new test confirming the existing behavior for this case, since it is so subtle. I.e.:

```
assertScoreByCode('26,46,47,51,53+51:46,53:46', 20, 'Collector does not score duplicated cards');
```